src/libse/Common/IfoParser.cs

```diff
@@ -39,37 +39,39 @@ public class VtsVobs
             public List<string> GetAllLanguages()
             {
                 var list = new List<string>();
-                for (int i = 0; i < Subtitles.Count; i++)
+                var minCount = Math.Min(Subtitles.Count, Math.Min(SubtitleIDs.Count, SubtitleTypes.Count));
```

Comment:
The loop only need to go to min between the three Subtitles,  SubtitleIDs and SubtitleTypes